src/compute-types/src/dataflows.rs

```diff
@@ -247,12 +247,20 @@ impl<P, S, T> DataflowDescription<P, S, T>
 where
     P: CollectionPlan,
 {
+    /// Identifiers of imported objects (indexes and sources).
+    pub fn import_ids(&self) -> impl Iterator<Item = GlobalId> + '_ {
+        self.index_imports
+            .keys()
+            .chain(self.source_imports.keys())
+            .copied()
+    }
+
     /// Identifiers of exported objects (indexes and sinks).
     pub fn export_ids(&self) -> impl Iterator<Item = GlobalId> + '_ {
         self.index_exports
             .keys()
             .chain(self.sink_exports.keys())
-            .cloned()
+            .copied()
```

Comment:
Is there an actual runtime difference here or were you just aligning the syntax of similar neighboring functions?

Reply:
For `Copy` types, both methods should do exactly the same thing. Using `copied` here just (a) makes the two functions consistent as you assumed and (b) makes it clearer that we are not doing any expensive cloning here.